lib/beautiful/xresources.lua

```diff
@@ -75,8 +75,8 @@ end
 -- @treturn number DPI value.
 function xresources.get_dpi(s)
     s = get_screen(s)
-    if dpi_per_screen[s] then
-        return dpi_per_screen[s]
+    if s then
+        return dpi_per_screen[s] or s.dpi
```

Comment:
If I query the DPI of some screen (which has no outputs), then the result will always be 96, since this is what the `get_dpi` function above returns. Shouldn't this rather use some of the calculations below instead if no outputs are available?
So, my proposal:
```lua
if dpi_per_screen[s] then
    return dpi_per_screen[s]
end
if #s.outputs > 0 then
    return s.dpi
end
```